localization/english/wc_traits_l_english.yml

```
@@ -489,4 +491,13 @@
  trait_dire_desc:0 "A small number of trolls are born larger and more powerful than their brethren. Those blessed with this form are called dire trolls, or berserkers, and they stand at the very top of the troll fighting elite."
  
  TRAIT_FLAG_DESC_return_to_twisting_nether:0 "Can return to [twisting_nether|E]"
- TRAIT_FLAG_DESC_makes_incapable:0 "@warning_icon!#X Makes [wc_incapable|E]#!"
\ No newline at end of file
+ TRAIT_FLAG_DESC_makes_incapable:0 "@warning_icon!#X Makes [wc_incapable|E]#!"
+ 
+ trait_lich_queen:0 "Lich Queen"
+ trait_lich_king_common_desc:0 "Originally created by Kil'jaeden from the spirit of the orc shaman Ner'zhul trapped inside the Helm of Domination. Lich King's unimaginable necromantic potential allows him to control hundreds of thousands of walking deads."
+ 
+ trait_in_ice_prison:0 "In Ice Prison"
```

In Ice Prison just doesn't sound right. Could also be changed to something like 'Frozen Throne' or 'Imprisoned in Icecrown', alternatively.
```suggestion
 trait_in_ice_prison:0 "Inside an Ice Prison"
```

---

It's a title, so should be without articles.
```suggestion
 trait_in_ice_prison:0 "Inside Ice Prison"
```